.github/actions/observe-build-status/action.yml

```diff
@@ -54,7 +54,7 @@ runs:
       id: get-build-duration-millis
       shell: bash
       run: |
-        duration=$(expr $(date +'%s') - $(date --reference "$GITHUB_ACTION_PATH" +"%s"))
+        duration=$(expr $(date +'%s') - $(date -r "$GITHUB_ACTION_PATH" +"%s"))
```

Comment:
Unrelated fix needed for MacOS runners that don't support `date --reference`, found due to CI health